pkg/task/identity_test.go

```diff
@@ -24,7 +24,36 @@ import (
 func TestNewTaskIdentifier(t *testing.T) {}
 
 // TODO
-func TestIsPod(t *testing.T) {}
+func TestIsPod(t *testing.T) {
+	tests := map[string]struct {
+		taskIdentifier taskIdentifier
+		isPod          bool
+	}{
+		"True": {
```

Comment:
Instead of putting `True` as a key.
It would make more sense to put key as for pod `kindPod#1` and for not pod  `kindNotPod#1`, `kindNotPod#2` and so on.

Reply:
Point taken.